cordova-sdk/src/android/OptimoveInitProvider.java

```diff
@@ -54,6 +54,11 @@ public boolean onCreate() {
         OptimoveConfig.Builder configBuilder = new OptimoveConfig.Builder(optimoveCredentials,
                 optimoveMobileCredentials);
 
+        if (optimoveMobileCredentials == null){
```

Comment:
What about the other permutation, if Optimove credentials null?

P.S. can use `===` ?

P.P.S. is formatting w/ Prettier set up?

Reply:
1. all the code below is only relevant for Optimobile. If optimove creds are null we can still run it. Then presume base sdk handles single credentials correctly
2. fixed formatting
3. added same for iOS